libraries/pipes/implementations/typescript/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "dagster_pipes_typescript",
```

Comment:
Thoughts on naming this `dagster-pipes` since it's already in the context of js/ts? That way someone can `npm i dagster-pipes`?

Are underscores or dashes considered best practice for for npm packages?

Reply:
I'm going to go ahead and merge, and then do a quick follow-up to rename the package, and apply our linting rules from the monorepo.

Really aesomes work, @shimon-cherrypick .